i18n-helpers/src/lib.rs

```diff
@@ -127,19 +129,44 @@ pub fn extract_events<'a>(text: &'a str, state: Option<State<'static>>) -> Vec<(
 }
 
 /// Markdown events grouped by type.
-#[derive(Debug, Copy, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
 pub enum Group<'a> {
     /// Markdown events which should be translated.
     ///
     /// This includes `[Text("foo")]` as well as sequences with text
     /// such as `[Start(Emphasis), Text("foo") End(Emphasis)]`.
     Translate(&'a [(usize, Event<'a>)]),
 
+    /// Owned version of `Group::Translate`.
+    TranslateOwned(Vec<(usize, Event<'a>)>),
```

Comment:
I think it would be okay to remove the borrowing variants and always own the `Event<'a>` values. My thinking is that `mdbook build` is something you run offline, so it's okay if it takes a few 100ms more (especially if it can simplify the code).

Reply:
Thanks for updating!